app/app.js

```diff
@@ -75,15 +89,23 @@ const init = () => {
 
         return;
     }
-    getData(values.parliament, values.term).then(data => {
-        const parties = data.reduce((acc, party) => {
+    getData(values.parliament, values.term, values.lang).then(data => {
```

Comment:
I suppose you just forgot to remove this `values.lang` here? It seems like we don't actually do anything with it.